web/src/styles/globals.css

```
@@ -1,7 +1,687 @@
+ * HeroUI doesn't already provide as utilities. Generates
+ * bg-background-alt / bg-surface-alt, text-link / text-link-hover,
+ * bg-attention & friends, bg-code-surface & friends, and
+ * shadow-elevation-sm/md/lg + shadow-modal.
+ *
+ * Components consume these the same way as HeroUI utilities;
+ * the @theme block here is non-overlapping with HeroUI's tokens.
+ * ============================================================= */
+@theme {
+  /* Brand font stacks — generate font-sans / font-mono / font-display
+   * utilities and back the typography roles + base resets below. */
+  --font-sans: "Mozilla Text", Arial, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Arial" to be "arial" (value-keyword-case)

(value-keyword-case)